### ISSUE TITLE
No more data in the tables after the last update

### ISSUE THREAD

**Issue by NL-CCMLM** [nextcloud/tables#1039]
### Steps to reproduce

Update the app (without saving data) 

### Expected behavior

The data shouldn't be removed but...

### Actual behavior

There is no more data in tables.
Gone.
Vanished.
Help ?

### Tables app version

0.6 > 0.7.1

### Browser

all

### Client operating system

W10 W11 Debian ...

### Web server

None

### PHP engine version

None

### Database

None

**Comment by FieldofClay** [comment]
I've had the same issue. Upgrade from 0.6->0.7.1 and all of the tables are empty. I've tried accessing via Chrome, Firefox and the Android app, all the same.

Server is ArchLinux, running nginx 1.25.5, php 8.2.18, mariadb 11.3.2.

I have looked at the DB and can see all of the rows still under the oc_tables_rows table.

Logs from the upgrade:
[nextcloud.log](https://github.com/nextcloud/tables/files/15182461/nextcloud.log)

**Comment by slu-010101** [comment]
Same here, all tables are empty!

**Comment by 90madness** [comment]
Yes, Same here. After Update to 0.7.1 all tables are empty. It's an absolute catastrophe. using Nextcloud All in One on Debian

**Comment by slu-010101** [comment]
This works for me:
https://github.com/nextcloud/tables/issues/999#issuecomment-2077578059

**Comment by tomtom** [comment]
Is there a chance this will be fixed? The proposed solution in #999 doesn't work for people where occ (and/or web-occ) doesn't work.

**Comment by hermann-san** [comment]
Just for People who are on Linux and don't know how to execute the occ command. This is run in the Nextcloud install directory, wherever this may be for you. For me it's /var/www/nextcloud
`sudo -u www-data php occ tables:legacy:transfer:rows --all`
This has worked for me

**Comment by tomtom** [comment]
There is also the web-occ app that could be of help here - but this doesn't
always work with every version of nc. Alternatively, one can use the php
script mentioned above.

**Comment by joshtrichards** [comment]
This is an older issue. Still experiencing this behavior today?

**Comment by NL-CCMLM** [comment]
Soooooorry, I should have marked the issue as [SOLVED], because the @slu-010101 was the solution for me, and the further updates were definitively OK

Thanks for your great job !
